decidim-debates/db/migrate/20201126112752_archive_debates.rb

```diff
@@ -1,8 +0,0 @@
-# frozen_string_literal: true
-
-class ArchiveDebates < ActiveRecord::Migration[5.2]
```

Comment:
I'm not so sure about just deleting this migration... Should we also do the reversed migration (like removing this column)? 
Mind that we actually don't officially support pointing to `develop` branch and this PR didn't get in any release, but there still could be someone that have this column somewhere (although it shouldn't be an issue also to have this column and not using it)

Reply:
Done in the latest commit!